lambdas/data-migration2/tests/test-index.js

```diff
@@ -153,21 +156,21 @@ test.serial('handler migrates executions, granules, files, and PDRs by default',
     1
   );
 
-  const granuleRecords = await t.context.granulePgModel.search(
+  const pdrRecords = await t.context.pdrPgModel.search(
     t.context.knex,
-    { granule_id: fakeGranule.granuleId }
+    { name: testPdr.pdrName }
   );
   t.is(
-    granuleRecords.length,
+    pdrRecords.length,
     1
   );
 
-  const pdrRecords = await t.context.pdrPgModel.search(
+  const granuleRecords = await t.context.granulePgModel.search(
```

Comment:
This is all great, but I wonder if we should verify that the `pdr_cumulus_id` on `granuleRecords[0]` matches the `pdrRecords[0].cumulus_id`? And use the `GranulesExecutionsPgModel.search` to determine if this a record joining the granule cumulus ID and execution cumulus ID?

Just to be sure that our migration code is doing the right thing and creating references from the granule to those other records

Reply:
That's a good point! I will update that to check

Reply:
[`94d2489` (#2402)](https://github.com/nasa/cumulus/pull/2402/commits/94d2489dff0ece950d4cbe1518a7052c2566e5f8)

Reply:
It seems like we didn't address this part:

> And use the GranulesExecutionsPgModel.search to determine if this a record joining the granule cumulus ID and execution cumulus ID?